d3b_utils/aws_bucket_contents.py

```diff
@@ -73,6 +77,10 @@ def fetch_bucket_obj_info(
         # ETag comes back with unnecessary quotation marks, so strip them
         object["ETag"] = object["ETag"].strip('"')
 
+        if full_filepath:
+            key = urlparse(object["Key"]).path
+            object["Filepath"] = f"s3://{bucket_name}{key}"
```

Comment:
I think this is missing a slash between bucket and key.

Reply:
No, `.path`'s return value starts with /

Reply:
i think it needs the slash now, since you aren't using urlparse

Reply:
No, I just cut out `source`. Is there a better way to do it without importing something?

Reply:
you shouldn't need to remove `source`. that is part of the prefix of most files. to return the url of a file in s3, all you need to do is :
```suggestion
            object["Filepath"] = f"s3://{bucket_name}/{key}"
```

Reply:
true fixed